packages/kbn-alerts-ui-shared/src/common/hooks/use_load_connectors.ts

```diff
@@ -27,6 +27,7 @@ export const useLoadConnectors = (props: UseLoadConnectorsProps) => {
   const { data, isLoading, isFetching, isInitialLoading } = useQuery({
     queryKey: ['useLoadConnectors', includeSystemActions],
     queryFn,
+    cacheTime: 0,
```

Comment:
What is the purpose of changing the cache time (garbage collection) to zero? Will this change affect other parts of the app?

Reply:
@JiaweiWu Any update on this?

Reply:
should be fine, nothing else is using this

Reply:
Yes, but why do we need to set it to `0`? What does it do and what issue does it solve?

Reply:
Ok, I spoke with @umbopepato, and help me understand. I made the hooks parameterized because they can be used in the future by other parts of the application and we may not want the caching to be set to zero.